geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -7191,6 +7191,7 @@ void updateStatsForGet(final RegionEntry regionEntry, final boolean hit) {
 
     final long now = cacheTimeMillis();
     if (regionEntry != null) {
+      logger.info("DEBUG: updateStatsForGet now=" + now);
```

Comment:
This logging should probably be removed.

Reply:
done